core/services/synchronization/stats_pusher.go

```diff
@@ -121,8 +128,14 @@ func (sp *statsPusher) Close() error {
 	}
 
 	gormCallbacksMutex.Lock()
-	sp.DB.Callback().Create().Remove(createCallbackName)
-	sp.DB.Callback().Update().Remove(updateCallbackName)
+	err := sp.DB.Callback().Create().Remove(createCallbackName)
+	if err != nil {
+		return err
+	}
+	err = sp.DB.Callback().Update().Remove(updateCallbackName)
+	if err != nil {
+		return err
```

Comment:
Maybe wrap them?

Reply:
converted them to just idiomatic if err := blah; err != nil ...